include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -25,6 +25,12 @@
 #include "parallel_backend_sycl_utils.h"
 #include "execution_sycl_defs.h"
 
+#define _ONEDPL_RADIX_WORKLOAD_TUNING 1
+//To achieve better performance number of segments and block size are variated depend on data size (32bits):
```

Comment:
Could you update the comment? block size is not used anymore. Could you also clarify the meaning of "(32bits)"? 
Also, can you replace "depend on" to "depending on"?

I guess the performance impact was evaluated only for 32-bit types according to the comment. What do you think about measuring it for other types (e.g. 16-bit and 64-bit wide ones)? Probably, the constant (128) may be different for them.

Reply:
32bits is more popular case I guess... But yes, it make sense to be estimated and probably to have another constant.

Reply:
I checked `count` kernel and found that the amount of allocated SLM does no depend on a input data type. So the performance difference for other input types is unlikely.